src/gengowatcher/browser_detector.py

```
@@ -0,0 +1,360 @@
+            self.logger.debug(f"Error getting Vivaldi UA: {e}")
+
+        # Fallback Vivaldi UA
+        return "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36 Vivaldi/7.8.3925.62"
+
+    def _get_chrome_ua(self) -> str:
+        """Get Chrome user agent string."""
+        try:
+            version = self._get_chrome_version()
+            if version:
+                return f"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/{version} Safari/537.36"
+        except:
```

Ruff (0.15.0)

[error] Do not use bare `except`

(E722)